test/draw_test.py

```diff
@@ -374,21 +374,144 @@ def todo_test_circle(self):
 
         self.fail()
 
-    def todo_test_polygon(self):
 
-        # __doc__ (as of 2008-08-02) for pygame.draw.polygon:
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+
+SQUARE = ([0, 0], [3, 0], [3, 3], [0, 3])
+DIAMOND = [(1, 3), (3, 5), (5, 3), (3, 1)]
+CROSS = ([2, 0], [4, 0], [4, 2], [6, 2],
+         [6, 4], [4, 4], [4, 6], [2, 6],
+         [2, 4], [0, 4], [0, 2], [2, 2])
+
+
+class DrawPolygonTest(unittest.TestCase):
+
+    def setUp(self):
+        self.surface = pygame.Surface((20, 20))
+
+    def test_draw_square(self):
+        pygame.draw.polygon(self.surface, RED, SQUARE, 0)
+        # note : there is a discussion (#234) if draw.polygon should include or
```

Comment:
This comment could go in the doc string too.

Reply:
done